test/compile/mark-tick.test.ts

```diff
@@ -12,33 +15,65 @@ describe('Mark: Tick', function() {
     assert.equal(tick.markType(), 'rect');
   });
 
+  function tickXY(moreEncoding = {}) {
+    const spec = {
+      'mark': 'tick',
+      'encoding': extend(
+        {
+          'x': {'field': 'Horsepower', 'type': 'quantitative'},
+          'y': {'field': 'Cylinders','type': 'ordinal'}
+        },
+        moreEncoding
+      ),
+      'data': {'url': 'data/cars.json'},
+    };
+    return spec;
+  }
+
   describe('with quantitative x', function() {
+    const model = parseModel({
+      'mark': 'tick',
+      'encoding': {'x': {'field': 'Horsepower', 'type': 'quantitative'}},
+      'data': {'url': 'data/cars.json'}
+    });
+
+    const props = tick.properties(model);
     it('should be centered on y', function() {
-      // TODO
+      assert.deepEqual(props.yc, {value: 10.5});
     });
 
     it('should scale on x', function() {
-      // TODO
+      assert.deepEqual(props.xc, {scale: X, field: 'Horsepower'});
     });
   });
 
   describe('with quantitative y', function() {
+    const model = parseModel({
+      'mark': 'tick',
+      'encoding': {'y': {'field': 'Cylinders','type': 'quantitative'}},
+      'data': {'url': 'data/cars.json'}
+    });
+
+    const props = tick.properties(model);
     it('should be centered on x', function() {
-      // TODO
+      assert.deepEqual(props.xc, {value: 10.5});
     });
 
     it('should scale on y', function() {
-      // TODO
+      assert.deepEqual(props.yc, {scale: Y, field: 'Cylinders'});
     });
   });
 
   describe('with quantitative x and ordinal y', function() {
+    const model = parseModel(tickXY());
```

Comment:
You don't really use the modeEncoding here. Let's just inline the model as in the other test cases.